src/gui/modals/SetupDialog.cpp

```diff
@@ -723,10 +724,18 @@ SetupDialog::SetupDialog(ConfigTabs tab_to_open) :
 		m_assignableMidiDevices->setCurrentIndex(current);
 	}
 
+	// MIDI Recording tab
+	auto* midiRecordingTab = new TabWidget(tr("Behavior when recording"), midi_w);
+	int mwCounter = 0;
+	addLedCheckBox(tr("Auto-quantize notes in Piano Roll"), midiRecordingTab, mwCounter,
```

Comment:
It doesn't look necessary to use a counter here because there's a single CheckBox but if we are using wouldn't it be more consistent to the rest of the code to reuse `counter` as well?
```suggestion
	counter = 0;
	
	// MIDI Recording tab
	auto* midiRecordingTab = new TabWidget(tr("Behavior when recording"), midi_w);
	addLedCheckBox(tr("Auto-quantize notes in Piano Roll"), midiRecordingTab, counter,
```